UnitySDK/Assets/ML-Agents/Scripts/Sensor/CameraSensorComponent.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace MLAgents.Sensor
 {
+    [AddComponentMenu("ML Agents/Sensors/Camera Sensor")]
```

Comment:
Intentionally left "Component" off the sensor menu items; I think it's a bit wordy otherwise but willing to change it.

Reply:
The Unity UI guidelines recommend against nesting menus.  We should just group the alike menu items using the `componentOrder` constructor variable.

Reply:
```suggestion
    [AddComponentMenu("ML Agents/Camera Sensor", kSensorMenuGroup)]
```
Where `kSensorMenuGroup` is defined somewhere.